Makefile

```diff
@@ -1,14 +1,18 @@
-# Include the Makefile from the ansible/awx-operator repository.
-# This Makefile is created with the clone-awx-operator.py script.
-include Makefile.awx-operator
```

Comment:
@miles-w-3 I've only been able to take a bit of a cursory look at these changes but if we no longer need the makefile from the awx-operator repo should we also update the `clone-awx-operator.py` script so it doesn't create it?

Reply:
yeah I think that would be fine. I removed the dependency on this makefile by adding in a few more variables. I think there's a chicken and egg problem on main right now, since the has stages that invoke clone-awx-operator.py, but the makefile can only run once Makefile.awx-operator has been created